draft-ietf-quic-manageability.md

```diff
@@ -905,6 +905,119 @@ need to be flow based.  However, 5-tuple (plus eventually connection ID if
 present) matching is still beneficial for QoS given all packets are handled by
 the same congestion controller.
 
+## QUIC and Network Address Translation (NAT)
+
+Network Address Translators (NATs) are a widely deployed means of multiplexing
+multiple private IP addresses over scarce IPv4 public address space by replacing
+those addresses and using ports to distinguish those connections. The new
+address can also guarantee that packets move through a proxy throughout the life
+of a connection, so that the connection can continue with the required state at
+that proxy.
+
+This section uses the colloquial term NAT to mean NAPT (section 2.2 of
+{{?RFC3022}}), which overloads several IP addresses to one IP address or to an
+IP address pool, as commonly deployed in carrier-grade NATs or residential NATs.
+QUIC Connection IDs are opaque byte fields that are expressed consistently
+across all QUIC versions {{QUIC-INVARIANTS}}. This feature may appear to present
+opportunities to optimize NAT port usage and simplify the work of the QUIC
+server. In fact, NAT behavior that relies on CID may instead cause connection
+failure when endpoints change Connection ID, and disable important protocol
+security features.
+
+The remainder of this section explains how QUIC supports NATs better than other
+connection-oriented protocols, why NAT use of Connection ID might appear
+attractive, and how NAT use of CID can create serious problems for the
+endpoints.
+
+The conclusion of this section is that NATs should retain their existing
+4-tuple-based operation and refrain from parsing or otherwise using QUIC
+connection IDs.
+
+{{?RFC4787}} contains some guidance on building NATs to interact constructively
+with a wide range of applications. This document extends the discussion to QUIC.
+
+### QUIC and NAT Rebinding
+
+By using the CID, QUIC connections can survive NAT rebindings as long as no
+routing function in the path is dependent on client IP address and port to
+deliver packets between server and NAT. Reducing the timeout on UDP NATs might
+be tempting in light of this property, but not all QUIC server deployments will
+be robust to rebinding.
+
+### The Lure of the Connection ID
+
+There are a few reasons that CID-aware NATs could seemingly appear attractive.
```

Comment:
editorial, like the comment above: please end this with a snarky conclusion that they are all wrong. :)

Reply:
Made obsolete by Mirja's suggestion

Reply:
overcome by events